packages/adapter-cloudflare/index.js

```diff
@@ -16,13 +16,27 @@ export default function (options = {}) {
 			builder.rimraf(tmp);
 			builder.mkdirp(tmp);
 
-			// generate 404.html first which can then be overridden by prerendering, if the user defined such a page
```

Comment:
Should we still account for user-defined 404.html pages before generating it for them? Right now we would be overwriting them.

Reply:
Yea it might be better to move this back about `builder.writePrerendered` line